content/en/docs/Reference/veneers.md

```diff
@@ -0,0 +1,397 @@
+---
+title: "Veneers"
+linkTitle: "Veneers"
+weight: 5
+date: 2022-06-30
+---
+
+File-Based Catalogs (FBC) are a major improvement to the imperative update graph approaches of previous versions. FBCs give operator authors a [declarative and deterministic approach to defining their update graph](https://olm.operatorframework.io/docs/concepts/olm-architecture/operator-catalog/creating-an-update-graph/). However, FBCs can get complex, especially as the number of releases and dependencies scale.
+
+Enter the `veneer`, a simpler transitional step to generating an FBC.
+
+## Design
+
+There are two components to every `veneer`:
+1. An arbitrary API
+2. An executable which processes #1 and produces a valid FBC.
+
+## Benefits
+
+- A set of consistent expectations from including as standard functionality with `opm`
+- A good set of use-cases for simplifying the ability of operator authors to manage FBCs, including contributing them to an index
+- A transitive benefit from consistency of practice as authors move from operator to operator
+
+The veneers which accompany `opm` are:
+- [`basic veneer`](#basic-veneer), which provides simplification of FBC concepts; and
+- [`semver veneer`](#semver-veneer), which provides the capability of sophisticated channel generation
+
+## Basic Veneer
+The `basic veneer` is an input schema which eliminates FBC information that can be retrieved from existing registries when we process it.
+Users provide all components of an [FBC schema](https://olm.operatorframework.io/docs/reference/file-based-catalogs/#olm-defined-schemas), but supply only the bundle image reference for any `olm.bundle` objects. This results in a greatly simplified, smaller document.
+
+
+In a very simple example, we define an `olm.package` and an `olm.channel` composed of two `olm.bundle` objects that have an image name attribute but no other attributes/properties.
+```yaml
```

Comment:
@kevinrizza I close my comment for 2 reasons:
a) It is clear to me now that the veener schema is only a helper to generate the FBC and it will not be used in another place
b) I still think that we are missing here better describes each case's motivations and what are the pros/cons of the results produced by both. However, I will do a better suggestion.